src/flamenco/fd_flamenco_base.h

```diff
@@ -15,6 +15,12 @@
 #define FD_FUNK_KEY_TYPE_ACC ((uchar)1)
 #define FD_FUNK_KEY_TYPE_ELF_CACHE ((uchar)2)
 
+/* CLUSTER_VERSION is the default value for the cluster version
+   in the epoch context. This value will foll forward to the
+   latest version.
+*/
+#define DEFAULT_CLUSTER_VERSION (2000)
```

Comment:
Could we call this `FD_DEFAULT_AGAVE_CLUSTER_VERSION`?

- Makes it clear that the version is the Agave version. Agave and Firedancer will have different versioning schemes.
- We should namespace everything with FD_